CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG ${CMAKE_CXX_FLAGS_DEBUG}")
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR
         CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
     set(WARNING_FLAGS "-Wall -Wextra -Wno-unused-parameter")
-    set(CMAKE_CXX_FLAGS "-std=c++0x ${WARNING_FLAGS} ${CMAKE_CXX_FLAGS}")
+    set(CMAKE_CXX_FLAGS "-std=gnu++11 ${WARNING_FLAGS} ${CMAKE_CXX_FLAGS}")
```

Comment:
I pushed some changes to Trunk which should fix the MinGW issues without needing to change the `-std=` flag.  You may need additional adjustments for a static build though, since it fixes the incorrect detection on the `PLASMA_DLL` defines too -- but I haven't tested that.